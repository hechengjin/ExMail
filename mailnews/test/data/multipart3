Content-Type: multipart/mixed; boundary=boundary

--boundary
Content-Type: text/html

<html>


<body>



Here, the HTML tags should be stripped out.



</body>



</html>







--boundary--
