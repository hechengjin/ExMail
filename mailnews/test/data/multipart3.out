Here, the HTML tags should be stripped out.